spec/classes/software_vcsscm_git_spec.rb

```diff
@@ -13,6 +13,134 @@
         else
           it { is_expected.to compile.with_all_deps }
           it { is_expected.to contain_package('git') }
+          it { is_expected.not_to contain_package('gitk') }
+          it { is_expected.not_to contain_package('git-gui') }
+          it { is_expected.not_to contain_package('bash-completion') }
+          it { is_expected.not_to contain_file('/opt/bash-git-prompt/') }
```

Comment:
@edestecd Is there a `contain_directory` function (I don't find any info on that on the Web), or should I simply remove the trailing slash?

Reply:
Seems like that is a vcsrepo and not a file: https://github.com/comsolit/puppet-software/blob/419484399474c4132350310b1bf186da9c839a4b/manifests/vcsscm/git.pp#L36

it { is_expected.not_to contain_vcsrepo('/opt/bash-git-prompt/') }

Reply:
Well, "not" doesn't seem to be the problem. But thanks for the hint, anyway. I think I start to understand the inner workings of RSpec.

Unfortunately, the `is_expected.to contain_file('/opt/bash-git-prompt')` still fails. I'm out of luck, out of ideas anyway. :confused: